script.js

```diff
@@ -0,0 +1,62 @@
+const song = document.querySelector('.song');
+const play = document.querySelector('.play');
+const outline = document.querySelector('.moving-outline circle');
+const video = document.querySelector('.video-container video');
+const sounds = document.querySelectorAll('.container__sounds button');
+const timeDisplay = document.querySelector('.time-display');
+const timeSelect = document.querySelectorAll('.time-select');
+const outlineLenght = outline.getTotalLength();
+const ZERO = '0';
+let fakeTime = 600;
+
+outline.style.strokeDasharray = outlineLenght;
+outline.style.strokeDashoffset = outlineLenght;
+
+const checkPlaying = song => {
+  if(song.paused) {
+    song.play();
+    play.src = 'svg/pause.svg';
+    video.play();
+  }else {
+    song.pause();
+    play.src = 'svg/play.svg';
+    video.pause();
+  }
+};
+
+play.addEventListener('click', () => checkPlaying(song));
+
+function addZero(n) {
+  return (parseInt(n, 10) < 10 ? ZERO : '') + n;
+}
+
+timeSelect.forEach(option => {
+  option.addEventListener('click', function(){
+    fakeTime = this.getAttribute('data-time');

```

Comment:
If you use data-... attributes, you can access the elements faster. Check the answers here: https://stackoverflow.com/questions/16719320/what-is-the-fastest-way-to-get-multiple-data-attributes-from-the-same-element

`.getAttribute(`) should be used as a fallback method for older browsers. Find the articles about getAttributes and dataset to decide what to use.

Reply:
DONE!